docs/Authentication.md

```diff
@@ -161,19 +161,19 @@ df.write
 ## Device Authentication
 If no authentication parameters were passed, the connector will request for user authentication by writing a token 
 to the console. This token can be used to authenticate at https://login.microsoftonline.com/common/oauth2/deviceauth 
-and will allow temporary access (about one hour). For Python use this [sample](../samples/src/main/python/pyKusto.py).
-The user needs appropriate privileges for the Kusto cluster as explained in [Kusto Sink authentication section](KustoSink.md#authentication). 
+and will allow temporary access (about one hour). For Python, use this [sample](../samples/src/main/python/pyKusto.py).
+The user needs appropriate privileges for the Kusto cluster as explained in [Kusto Sink authentication section](KustoSink.md#authentication).
 
 ### Device Authentication for PySpark
 Please refer to the [Python samples](../samples/src/main/python/pyKusto.py).
 
->**Note:** Device authentication is not recommended for production   
+>**Note:** Device authentication is not recommended for production
 
 ## Authentication via token provider callback
 
-User can provide a class to be used as a callback for providing an AAD token for authentication. 
+User can provide a class to be used as a callback for providing an AAD authentication token. 
 The class should have a constructor that accepts one argument of type CaseInsensitiveMap[String] (which will contain
-the options provided to the connector). The class should extend Callable<String> with Serializeable and is expected
+the options provided to the Connector). The class should extend Callable<String> with Serializeable and is expected
```

Comment:
My thought process is based on the fact that "Connector" is short for "Azure Data Explorer Connector", which is a proper noun, and thus all parts are capitalized.

Good catch - it can go either way, though.